tests/components/cast/test_media_player.py

```diff
@@ -282,6 +271,62 @@ def discover_chromecast(service_name: str, info: ChromecastInfo) -> None:
     assert add_dev2.call_count == 1
 
 
+async def test_manual_cast_chromecasts(hass):
+    """Test only wanted casts are added for manual configuration."""
+    cast_1 = get_fake_chromecast_info(host="configured_host")
+    cast_2 = get_fake_chromecast_info(host="other_host", uuid=FakeUUID2)
+
+    # Manual configuration of media player with host "configured_host"
+    discover_cast, add_dev1 = await async_setup_cast_internal_discovery(
```

Comment:
These tests should preferably be rewritten to not interact directly with integration code but set up the component and corresponding platform and let it add an entity.